python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/ip-name-server-groups.j2

```diff
@@ -0,0 +1,35 @@
+{#
+ Copyright (c) 2023-2024 Arista Networks, Inc.
+ Use of this source code is governed by the Apache License 2.0
+ that can be found in the LICENSE file.
+#}
+{% for name_server_group in ip_name_server_groups | arista.avd.natural_sort("name") %}
```

Comment:
please add a documentation templates for `ip name-server groups` listing the servers inside

Reply:
Added.